src/main.cpp

```diff
@@ -2131,7 +2131,7 @@ void setup() {
     previousMillisDisplay = currentTime;
     previousMillisMQTT = currentTime;
     previousMillisInflux = currentTime;
-    previousMillisVoltagesensorreading = currentTime;
+    previousMillisOptocouplerreading = currentTime;
```

Comment:
```suggestion
    previousMillisOptocouplerReading = currentTime;
```

Reply:
Wasn't sure about that because everywhere I saw a var with reading at the end was writen small.
Going to change it, thanks!

Reply:
I think the rule of thumb is that we want to have a capital letter whenever a new word starts